src/main/java/frc/robot/common/ControlPanelColor.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.common;
+
+import edu.wpi.first.wpilibj.util.Color;
+
+public enum ControlPanelColor {
+  R(new Color(0.325, 0.453, 0.221)),
+  Y(new Color(0.269, 0.550, 0.181)),
+  G(new Color(0.175, 0.469, 0.356)),
+  B(new Color(0.188, 0.503, 0.310)),
```

Comment:
Renaming these to match the DS doesn't reduce readability since it's in a class for colors and RYGB can be easily understood to mean red, yellow, green, blue. It makes it easier to parse the ds color since it's one of these values.

`ControlPanelColor.valueOf("R")` is valid